src/xcode/ENA/ENA/Source/Scenes/Home/Cells/Statistics/HomeLinkCardView/HomeLinkCardViewModel.swift

```diff
@@ -0,0 +1,92 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import UIKit
+import OpenCombine
+
+class HomeLinkCardViewModel {
+	
+	// MARK: - Init
+
+	init(for linkCard: SAP_Internal_Stats_LinkCard) {
+		switch HomeLinkCard(rawValue: linkCard.header.cardID) {
+		case .pandemicRadar:
+			setupPandemicRadar(for: linkCard)
+		case .none:
+			Log.info("Link card ID \(linkCard.header.cardID) is not supported", log: .ui)
+		}
+	}
+	
+	// MARK: - Internal
+	
+	@OpenCombine.Published private(set) var title: String?
```

Comment:
They all are private, better to put them under private mark

Reply:
Setter is private, Getter is internal. I would keep it under internal, cause you can see it from outside